src/test/java/org/scijava/module/ModuleServiceTest.java

```diff
@@ -34,8 +34,12 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertSame;
 
+import java.security.GeneralSecurityException;
+
+import org.junit.Assert;
```

Comment:
The typical way to do it is:
```
import static org.junit.Assert.assertEquals;
```
And then below, just `assertEquals` rather than `Assert.assertEquals`.